tests/test_project_time.py

```diff
@@ -78,38 +78,43 @@ def test_convert_to_project_time(registry_mgr):
         raw_data=tempo_data_with_tz,
         converted_data=tempo_data_mapped_time,
     )
-    compare_time_conversion(
-        resstock_time_dim, project_time_dim, wrap_time=False, expect_error=True
-    )
-    compare_time_conversion(
-        comstock_time_dim, project_time_dim, wrap_time=False, expect_error=True
-    )
-    compare_time_conversion(
-        resstock_time_dim, project_time_dim, wrap_time=True, expect_error=False
-    )
-    compare_time_conversion(
-        comstock_time_dim, project_time_dim, wrap_time=True, expect_error=False
-    )
+
+    # [2.1] test time-wrap when dataset and project have different time interval type
```

Comment:
It looks like you could fairly easily split these into different tests because the setup is simple. Then the comments become test names. The main advantage is that you don’t have to worry about resetting state. Not sure if it makes sense, but you could also parameterize things like time zone and wrap_time.